docker-compose-ci.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.3"
```

Comment:
Why can't we reuse the existing `docker-compose.yml` and update it as necessary? Should make it extensible via environment variables or configuration so that it can be reused by both developers and CI.

Reply:
We extended the PackageSuiteRunner implementation to accept arguments and we're reusing the docker-compose.yml.